core/src/mill/util/OS.scala

```diff
@@ -0,0 +1,5 @@
+package mill.util
+
+object OS {
+  val isWindows: Boolean = System.getProperty("os.name").toLowerCase.startsWith("windows")
```

Comment:
Do we need this if we already have `scala.util.Properties.isWin`? Same for the version of this in `build.sc`

(Note that we should not use `scala.util.Properties.isWin` in the Mill client, since that only uses Java APIs to avoid classloading/initialization slowness)

Reply:
Somehow that escaped me. I'll change to use Scala's `isWin` except on the client.